_scripts/import_resources.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+import os
+import pickle
+import datetime
+
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+
+TOKEN_FILE = '.google_oauth_token'
+SCOPES = ['https://www.googleapis.com/auth/spreadsheets.readonly']
+RESOURCES_SHEET_ID = '1QSQgxceR8BL03qsR-GY8bw4tsdsx30kovU7wVXPUcxo'
+RANGE = 'Form Responses 1!A2:J'
+
+COL_TIMESTAMP = 0
+COL_SUBMITTER = 1
+COL_MEETS_STANDARDS = 2
+COL_CATEGORY = 3
+COL_COUNTRY = 4
+COL_URL = 5
+COL_NAME = 6
+COL_DESCRIPTION = 7
+COL_STATE = 8
+COL_VERIFIED_BY = 9
+# TODO: we could write integrated_timestamp into the spreadsheet if
+# we want to have a record of this action.
+ENABLE_VERIFICATION = False
+
+
+def is_approved(row):
+    if not ENABLE_VERIFICATION:
+        return True
+    if COL_VERIFIED_BY >= len(row) or not row[COL_VERIFIED_BY]:
+        return False
+    return True
+
+
+def make_row_filename(row):
+    """Constructs standardized base filename for this resource.
+    
+    We will use YYYY-MM-DD of the submission timestamp followed by first 25
+    characters of the resource title with spaces replaced with underscores.
+    """
+   sanitized_name = row[COL_NAME].lower().replace(" ", "_")[:25]
```

Comment:
I'd replace any non `a-zA-Z0-9` character by a `_` - you don't want to get an exception if a title says "Covid-19: 1/3 of 60+ is at risk".